using_images/s2i_images/dot_net_core.adoc

```diff
@@ -33,31 +33,40 @@ and Python from within {product-title}.
 [[dot-net-core-supported-versions]]
 == Supported Versions
 
+* .NET Core version 2.0
 * .NET Core version 1.0
 * .NET Core version 1.1
 * Supported on Red Hat Enterprise Linux (RHEL) 7
```

Comment:
Is it intentional that '* Supported on Red Hat Enterprise Linux (RHEL) 7' is a bullet point in this list?